src/client/components/ShoppingCart/ModifyProductQuantity.js

```diff
@@ -3,31 +3,65 @@ import PropTypes from 'prop-types';
 import './ShoppingCart.styles.css';
 
 function ModifyProductQuantity(props) {
```

Comment:
It might be easier to understand the props a component take at a glance if you destructure them in the arguments. Would look like this:

```suggestion
function ModifyProductQuantity({product, shoppingCart, ...etc... }) {
```